<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Imponibile - Biagio Raucci</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>337</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="lE_imponibile">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="6">
     <widget class="QLabel" name="l_errore">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="5">
     <widget class="QLineEdit" name="lE_totFattura">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Imponibile:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="pB_calcola">
      <property name="text">
       <string>Calcola</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Totale fattura:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Aliquota Iva:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lE_aliquota">
      <property name="text">
       <string>22</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLineEdit" name="lE_iva">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>iva:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pB_calcola</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>pB_valutaClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lE_totFattura</sender>
   <signal>returnPressed()</signal>
   <receiver>Form</receiver>
   <slot>pB_valutaClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lE_aliquota</sender>
   <signal>returnPressed()</signal>
   <receiver>Form</receiver>
   <slot>pB_valutaClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pB_valutaClick()</slot>
 </slots>
</ui>
